tutorials/2d/2d_sprite_animation.rst

```diff
@@ -85,6 +85,28 @@ released.
         else:
             $AnimatedSprite.stop()
 
+ .. code-tab:: csharp
+    
+    using Godot;
+    using System;
+    
+    public class Kinematic2DSprite : KinematicBody2D
+    {
+        public override void _Process(float delta)
+        {
+            AnimatedSprite animatedSprite = (AnimatedSprite) GetNode("AnimatedSprite");
```

Comment:
It might be better to teach about `GetNode<T>` here? Also, we usually use `var` if the type is explicit on the right-hand side.
```suggestion
            var animatedSprite = GetNode<AnimatedSprite>("AnimatedSprite");
```